### PR TITLE
Chore/daef 525 Update to latest mobx-react-form

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR introduces latest available update of MobX React Form plugin.
Biggest change was the way `file` type fields are handled as from now you have to use `set` method in place of `onChange` in order to update field's value (e.g. on file drop or selection).

### Note
As a part of this PR a fix for `isValidAddress` endpoint regression issue has been implemented: https://issues.serokell.io/issue/CSM-557 (endpoint started to throw an error if invalid address has been passed to it - where in fact it should return success with boolean value `false`)

### CODE REVIEW

**Comment by DominikGuzei** [comment]
Reviewing this now

File: app/components/wallet/file-import/WalletFileImportDialog.js [hunk 1]
```diff
@@ -215,7 +215,9 @@ export default class WalletFileImportDialog extends Component {
           <FileUploadWidget
             {...walletFile.bind()}
             selectedFile={walletFile.value}
-            onFileSelected={walletFile.onChange}
+            onFileSelected={(file) => {
+              walletFile.set(file);
+            }}
```
Comment (DominikGuzei) [inline app/components/wallet/file-import/WalletFileImportDialog.js]: Why not keep it simple as before? `onFileSelected={walletFile.set}` 😉

Comment (nikolaglumac) [reply app/components/wallet/file-import/WalletFileImportDialog.js]: Tried that but it does not work... Believe it or not ;)

Comment (nikolaglumac) [reply app/components/wallet/file-import/WalletFileImportDialog.js]: Tried that but it does not work... Believe it or not ;)

Comment (DominikGuzei) [reply app/components/wallet/file-import/WalletFileImportDialog.js]: Ah ok … so probably the method is not bound then 😉  then put a comment there so we know for future reference 👍

Comment (nikolaglumac) [reply app/components/wallet/file-import/WalletFileImportDialog.js]: @DominikGuzei comment added - hope it is fine now ;)

**Pull request merged**